DuggaSys/sectioned.js

```diff
@@ -3482,8 +3482,9 @@ function validateForm(formid) {
 // This method is to be used to check if a code example should re-fetch the
 // contents of a code example based on eventual changes in external github-repo
 //------------------------------------------------------------------------------
-function refreshCodeExample(){
-  console.log("Should try to refresh a code example (check if re-fetching from external github repo is necessary")
+async function refreshCodeExample(exampelid) {
```

Comment:
The only change in sectioned.js from this PR was changing the refreshCodeExample function, rest of the code seems to be removing identation and readding it again.

Reply:
Good naming conventions and the comments seems explainatory of the function. A function to check if code examples nees to be refreshed using HTTP via AJAX. exampleid